src/router-middleware.js

```diff
@@ -26,8 +26,8 @@ export default function routerMiddleware($state) {
       return next({
         type: STATE_CHANGE_SUCCESS,
         payload: {
-          currentState: $state.current,
-          currentParams: $state.params,
+          currentState: $state.$current.name,
+          currentParams: $state.$current.params,
```

Comment:
Apologies if I'm missing something here but this doesn't seem to pass the params through correctly. Replacing line 30 with `currentParams: $state.params,` passes the params object as expected.